raiden-dapp/src/locales/en.json

```diff
@@ -1,28 +1,28 @@
 {
   "address-input": {
     "input": {
-      "placeholder": "Enter an address or ens name..."
+      "placeholder": "Enter an address or ENS name..."
     },
     "blockie-alt": "Selected token address blockie",
     "error": {
-      "invalid-address": "The input doesn't seem like a valid address or ens name",
-      "no-checksum": "The address is not in checksum format",
-      "empty": "The address cannot be empty",
-      "ens-resolve-failed": "Could not resolve an address for {url}"
+      "invalid-address": "Please enter a valid address or ENS name",
+      "no-checksum": "Please enter an address in checksum format. You can convert your address on https://ethsum.netlify.com",
+      "empty": "Please enter an address or ENS name",
+      "ens-resolve-failed": "Please enter a resolvable ENS name"
```

Comment:
If we are gonna remove a parameter from a resource, we should also remove it from the template.

Reply:
Sorry which param are you referring to?

Reply:
The `ens-resolve-failed` originally had a `{url}` parameter. This parameter is passed to the localization function in the template.

Reply:
Ah thanks good catch, will fix. Although I think they're just ignored, if no placeholders are present